doc/api/esm.md

```diff
@@ -33,15 +33,14 @@ node --experimental-modules my-app.mjs
 ### Supported
 
 Only the CLI argument for the main entry point to the program can be an entry
-point into an ESM graph. In the future `import()` can be used to create entry
-points into ESM graphs at run time.
+point into an ESM graph. `import()` can also be used to create entry points
```

Comment:
Non-blocking nit: I know this was in the original text, but `import` is a statement and not a function so the `()` should be removed. Same on line 43.

Reply:
should I just say "dynamic import" then?

Reply:
Maybe just leave it alone. (Hey, I said it was non-blocking!) I see that even the TC39 proposal uses `import()` and refers to it as "function-like". https://github.com/tc39/proposal-dynamic-import